packages/trpc/server/routers/viewer/eventTypes/update.handler.ts

```diff
@@ -496,6 +504,50 @@ export const updateHandler = async ({ ctx, input }: UpdateOptions) => {
     }
   }
 
+  if (deletedWebhooks && deletedWebhooks.length > 0) {
```

Comment:
If the child event type has other webhooks, then deleting the webhook on the parent deletes the wrong webhooks on the child event type

Reply:
I'm wondering if this would also happen in `packages\features\ee\managed-event-types\lib\handleChildrenEventTypes.ts:250`